src/helm-frontend/src/services/getDisplayRequestsByName.ts

```diff
@@ -9,7 +9,7 @@ export default async function getDisplayRequestssByName(
   try {
     const displayRequest = await fetch(
       getBenchmarkEndpoint(
-        `/benchmark_output/runs/${getBenchmarkSuite()}/${runName}/display_requests.json`,
+        `${getVersionBaseUrl()}/${runName}/display_requests.json`,
```

Comment:
note this doesn't really fix anything- so I can remove it from this PR.

Reply:
Acknowledged. I also just realized that the function name `getDisplayRequestssByName` is misspelled...